.nvmrc

```diff
@@ -0,0 +1 @@
+12.20.10
```

Comment:
Is a very specific version required? or is any version 12 o.k.? (if the latter, please change to just `12`)